Projects/Base/Data/RoboTile/BlueBrinstar.lua

```diff
@@ -4,10 +4,10 @@ tile_air = 0x0FF
 tile_unknown = 0x0DF
 tile_unknown_solid = 0x5F
 tile_interior = 0x81
-tile_bottom_edge = 0x121
-tile_right_edge = 0x121
-tile_bottom_right_outside_corner = 0x121
-tile_bottom_right_inside_corner = 0x121
+tile_bottom_edge = 0x141
```

Comment:
Sorry about that, these were supposed to be 0x1AA. Just merged a fix for that.

One thing to note is that 0x141 has high priority while all the tiles filled in the robo-tiling script have low priority. The tile 0x1AA also has a brighter blue part which I think connects better with the slopes